services/core-web/src/components/mine/Permit/MinePermitTable.js

```diff
@@ -158,7 +159,26 @@ const columns = [
               />
               Edit permit status
             </button>
-          </Menu.Item>
+          </Menu.Item>{" "}
+          <AuthorizationWrapper isMajorMine={text.major_mine_ind}>
+            <div className="custom-menu-item" key="3">
```

Comment:
Should this also have the "permit_create" permissions?

Reply:
I've actually removed it as it was unnecessary as the parent as the same auth requirements.